Online-Messaging-App/src/app/home/header/header.component.html

```diff
@@ -1,8 +1,16 @@
 <header>
     <mat-toolbar class="navbar">
         <div>Streamline Athletes</div>
+
         <div *ngIf="userLoggedIn">
+            <div class="headInfo">Welcome {{user[0].username}}!</div>
```

Comment:
Remove the "Welcome". Also, if you include `private auth: AuthenticationService` in the constructor, you can simply call `this.auth.getAuthenticatedUser().getUsername()` here, no need to pull in via an http request.

Reply:
done